### DISCUSSION TITLE
Interstitial ads,  Add a way to show close button after some seconds in viewing the ad

### DISCUSSION THREAD

**Discussion by Joshmatjjen** [invertase/react-native-firebase#4453] [Ideas] [answered]
Is there a way to implement showing close button after some seconds of viewing interstitial ad built for the Display & Video 360 environment... Like RewardAd

**Comment by mikehardy** [comment, marked as answer]
You may have better luck with an admob-specific module, we do offer admob here but we may actually discontinue it. If you'll notice there are no firebase-specific admob APIs in the underlying SDKs (firebase-ios-sdk and firebase-android-sdk) - obviously as a google product based on the same installation id (used for analytics analysis, messaging, app delivery, A/B testing, and admob) it is complementary - but it is definitely not the emphasis of this package and is not a site of any active development at the moment
